app/services/data_migrations/backfill_feedback_form_complete.rb

```diff
@@ -0,0 +1,19 @@
+module DataMigrations
+  class BackfillFeedbackFormComplete
+    TIMESTAMP  = 20230612071351
+    MANUAL_RUN = false
+
+    def change
+      applications.find_each(batch_size: 500) do |batch|
+        batch.update_columns(feedback_form_complete: true)
+      end
+    end
+
+  private
+
+    def applications
+      ApplicationForm.where.not(feedback_satisfaction_level: nil)
+                     .or(ApplicationForm.where.not(feedback_suggestions: nil))
+    end
+  end
```

Comment:
I'd add a clause to ignore the `feedback_form_complete: true` just in case you need to re-run this migration (if fails midway for example the migration will have to update the 90k records again). Let me know your thoughts :)

Reply:
good shout @tomas-stefano